qshift/templates/quarkus-application/skeletons/quarkus-orm-rest/src/main/resources/application.properties

```diff
@@ -1,6 +1,12 @@
 {% if values.database == 'quarkus-jdbc-postgresql' %}
-%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://${{ values.component_id }}-db:5432/${{values.component_id}}-db
-%prod.quarkus.datasource.username=postgres
+# Kubernetes Service name to access the DB should be as defined within the helm chart => postgresql:5432
+# apiVersion: v1
+# kind: Service
+# metadata:
+#  name: {{ include "postgresql.primary.fullname" . }}
+#  see: https://github.com/bitnami/charts/blob/9ba6ba51cf7c094e457396c423f5843df46023a2/bitnami/postgresql/templates/primary/svc.yaml#L4
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://${{ values.database_name }}-postgresql:5432/${{ values.database_name }}
+%prod.quarkus.datasource.username=${{ values.database_username }}
 %prod.quarkus.datasource.password=${{ values.database_password }}
```

Comment:
We should not pass the password to the application.properties

Reply:
Fixed: d84d675